### ISSUE TITLE
yowsup died again?

### ISSUE THREAD

**Issue by Whomakes** [tgalal/yowsup#3174]
Registration not working! The request is sent but the account authorization fails. 

D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate - MixHash(prologue)
DEBUG:dissononce.processing.impl.handshakestate:MixHash(prologue)
INFO:transitions.core:Finished processing state handshake exit callbacks.
INFO:transitions.core:Finished processing state handshake enter callbacks.
INFO:transitions.core:Finished processing state write_e exit callbacks.
INFO:transitions.core:Finished processing state read_e_ee_s_es enter callbacks.
I 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate - WriteMessage(payload, message_buffer)
INFO:dissononce.processing.impl.handshakestate:WriteMessage(payload, message_buffer)
D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate -     Processing token 'e'
DEBUG:dissononce.processing.impl.handshakestate:    Processing token 'e'
D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate -         e=GENERATE_KEYPAIR()
DEBUG:dissononce.processing.impl.handshakestate:        e=GENERATE_KEYPAIR()
D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate -         message_buffer.append(e.public_key)
DEBUG:dissononce.processing.impl.handshakestate:        message_buffer.append(e.public_key)
D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate -         MixHash(e.public_key)
DEBUG:dissononce.processing.impl.handshakestate:        MixHash(e.public_key)
D 2022-10-26 12:46:56,213 dissononce.processing.impl.handshakestate -     buffer.append(EncryptAndHash(payload))
DEBUG:dissononce.processing.impl.handshakestate:    buffer.append(EncryptAndHash(payload))
D 2022-10-26 12:46:56,764 yowsup.layers.network.dispatcher.dispatcher_asyncore - handle_close
DEBUG:yowsup.layers.network.dispatcher.dispatcher_asyncore:handle_close
D 2022-10-26 12:46:56,764 yowsup.layers.network.layer - Disconnected
DEBUG:yowsup.layers.network.layer:Disconnected
INFO:transitions.core:Finished processing state handshake exit callbacks.
INFO:transitions.core:Finished processing state init enter callbacks.

**Comment by houjianwen123** [comment]
same , when I start any demo client , it show connected to whats app server then disconnect at once..

"yowsup.demos.echoclient.layer - Connected to WhatsApp servers
yowsup.layers.network.dispatcher.dispatcher_asyncore - handle_close
yowsup.layers.network.layer - Disconnected
yowsup.layers.axolotl.layer_control - Disconnected, reboot_connect? = False"

**Comment by madduck** [comment]
Yeah, same thing here, it worked yesterday. It's like trying to nail pudding to the wall.

**Comment by dazzzl** [comment]
I tried it with protocol 4.1, but it does also not work. The error message changed, I think due to different token directory

**Comment by dreascomputer** [comment]
Seems to be working now

**Comment by dazzzl** [comment]
@dreascomputer  My yowsup is not working.... which env? which endpoints?

**Comment by jowiemann** [comment]
Works for me with:

    _MD5_CLASSES = "jJYkDZJ+SGfBMvlEeGKmSw=="

    _VERSION = "2.22.19"

Jörg

**Comment by dreascomputer** [comment]
On my side:
```
_VERSION = "2.22.22"
_MD5_CLASSES = "ObJxY0JKvPdBcLeq6zdqKw=="
```
Endpoints default but via hosts file redirected:
```
157.240.28.55 e1.whatsapp.net e15.whatsapp.net
34.194.255.230 e2.whatsapp.net e16.whatsapp.net
31.13.66.51 e3.whatsapp.net
31.13.71.50 e4.whatsapp.net
31.13.72.48 e5.whatsapp.net
31.13.79.54 e6.whatsapp.net
31.13.85.49 e7.whatsapp.net
31.13.65.50 e8.whatsapp.net
31.13.87.48 e9.whatsapp.net
108.168.254.65 e10.whatsapp.net
31.13.66.51 e11.whatsapp.net
31.13.70.50 e12.whatsapp.net
31.13.83.51 e13.whatsapp.net
31.13.92.48 e14.whatsapp.net
```
but it wasn't working first. I wait a couple of hours and it came back, seems to be connection problems like yesterday on app itself

**Comment by yanzhidiyi** [comment]
有人解决了这个问题没？请联系我 有红包酬谢

**Comment by reversePublic** [comment]
> 有人解决了这个问题没？请联系我 有红包酬谢

需要技术支持可以联系我 telegram:@tgforme2
